app/assets/javascripts/app/merge_orders.coffee

```diff
@@ -0,0 +1,30 @@
+class window.MergeOrder
+  constructor: (@$form) ->
+    # empty body
+
+  initTimeBasedServices: ->
+    return unless @$form.length
+
+    @$quantity_field = @$form.find(".js--edit-order__quantity")
```

Comment:
I'm having some trouble following your variables here. Can we change timed_quantity_display_field to duration_display_field for clarity?